D1673/blas_interface.md

```diff
@@ -1464,23 +1488,55 @@ private:
 #### `accessor_conjugate`
 
 The `accessor_conjugate` Accessor makes `basic_mdspan` return a
-`conjugated_scalar`.
+`conjugated_scalar` if the scalar type is `std::complex`.
```

Comment:
Change other wording that forbids use of conjugate view if the element type is not complex.